debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/oracle/generated/PlSqlParser.g4

```diff
@@ -2157,7 +2157,239 @@ access_driver_type
 
 external_table_data_props
     : DEFAULT DIRECTORY quoted_string
-    | LOCATION
+    | ACCESS PARAMETERS '(' et_oracle_loader | et_oracle_datapump | et_oracle_hdfs_hive | USING CLOB subquery ')'
+    | LOCATION '(' et_directory_spec? et_location_specifier (',' et_directory_spec? et_location_specifier )* ')'
+    ;
+
+et_oracle_loader
+    : comments_oracle_loader? record_format_info? field_definitions? column_transforms?
+    ;
+
+et_oracle_datapump
+    : ENCRYPTION (ENABLED | DISABLED)
+      | ( NOLOGFILE | LOGFILE et_directory_spec? et_file_spec )
+      | COMPRESSION ( ENABLED (BASIC |LOW | MEDIUM | HIGH)? | DISABLED)?
+      | HADOOP_TRAILERS (ENABLED | DISABLED) VERSION (COMPATIBLE | LATEST | quoted_string )
+      | comments_oracle_datapump
+    ;
+
+et_oracle_hdfs_hive
```

Comment:
Each of these rules need to chain into an optional follow-up `et_oracle_hdfs_hive` rule by adding `et_oracle_hdfs_hive*` to each of these rules.